### DISCUSSION TITLE
Using custom command

### DISCUSSION THREAD

**Discussion by JustNao** [community/community#23823] [New to GitHub]
<p>Hi !</p>
<p>I’ve recently started to search for something that allows me to download chapters from a website, and to convert into epub files. A <a href="https://github.com/TUSF/RRL2epub" rel="nofollow noopener">GitHub repositery</a> corresponded prefectly to my intent, so i tried using it. While i know the basics in programmation and can (now) work my way around in GitHub as a beginner, there is still an issue with it. I need to use a custom command, in my case “./RRL2epub rrl:1439”, but Git Bash keeps displaying “command not found”. I did clone the repositery and then execute Git Bash while being inside the folder, but to no avail. I would greatly appreciate if someone could point me the correct way to use it :slight_smile: !</p>
<p>Thanks in advance</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>If you’re asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:</p>
<ol>
<li>Check the README for instructions on how to operate the software, pointers to documentation or troubleshooting info</li>
<li>Check the <a href="https://help.github.com/articles/adding-support-resources-to-your-project/" rel="nofollow noopener">SUPPORT file</a>, if one exists</li>
<li>Check the <a href="https://help.github.com/articles/setting-guidelines-for-repository-contributors/" rel="nofollow noopener">CONTRIBUTING guide</a>, if one exists</li>
</ol>
<p>All of these documents can be found, if they exist, in the repository where you found the software itself.</p>
<p>I hope that helps!</p>

**Comment by lee-dohm** [comment]
<p>A post was split to a new topic: <a href="https://github.community/t/help-with-epub-and-avi-videos/115677">Help with epub and avi videos</a></p>
